static/src/stylesheets/module/nav/_navigation.scss

```diff
@@ -737,6 +737,11 @@ $c-navigation-expandable-background: colour(neutral-1);
             min-width: $navigation-toggler-width;
         }
     }
+
+    .is-not-modern .l-footer & {
+        min-width: $navigation-toggler-width;
+        font-size: 24px;
```

Comment:
I would usually but this is a magic number to get the icon to line up correctly.